packages/@core.keystone/setup.utils.js

```diff
@@ -2,6 +2,10 @@ const { v5: uuidv5 } = require('uuid')
 
 const { KnexAdapter } = require('@keystonejs/adapter-knex')
 const { MongooseAdapter } = require('@keystonejs/adapter-mongoose')
+const IORedis = require('ioredis')
+const session = require('express-session')
+const { COOKIE_MAX_AGE } = require('@condo/domains/user/constants/common')
```

Comment:
you don't need to extract a const if you don't know any reusable case for this const